dd-sdk-android/src/test/kotlin/com/datadog/android/log/internal/user/DatadogUserInfoProviderTest.kt

```diff
@@ -65,4 +70,68 @@ internal class DatadogUserInfoProviderTest {
         // Then
         verify(mockWriter).write(userInfo)
     }
+
+    @Test
+    fun `𝕄 keep existing properties 𝕎 setExtraProperties() is called`(
+        @Forgery userInfo: UserInfo
+    ) {
+        // When
+        val originalProperties = mapOf(
+            "key1" to 1,
+            "key2" to "one",
+        )
+        val newProperties = mapOf(
+            "key3" to 1.0,
```

Comment:
Here we're hardcoding the test fixtures, which makes the test unreliable. It'd be nice to use forgeries for the keys and values instead

Reply:
makes sense! I have not used the Elmyr library before, but I looked around for usage. I also had to add a string forgery factory to provide the type `Map<String,Any?>` - please let me know if that is incorrect